app/lib/account_deletion_config.rb

```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+module AccountDeletionConfig
+  module_function
+
+  def config
+    @config ||= load_config
+  end
+
+  def valid?
+    @valid ||= check_validity
+  end
+
+  def load_config
+    config = ThreeScale.config.ttl.account_deletion.each_with_object({}) do |(key, value), collection|
```

Comment:
OK I expected better something like `ThreeScale.config.account_deletion.each_with_object` and with keys:

`%i[suspension_period inactivity_period contract_unpaid_time]` or similar

Reply:
Sorry but I am not understanding you @hallelujah 😞

Reply:
I mean instead of having a ttl file for all features, I prefer a features file where you put configuration.
A specific TTL file is too detailed.

Reply:
aaaaah, ok, thanks for clarifying 😄 I though this comment was about something else... 😄 then let's keep the conversation in https://github.com/3scale/porta/pull/574#discussion_r254598187 😄